packages/drivers/routerlicious-driver/src/storageContracts.ts

```diff
@@ -0,0 +1,99 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as git from "@fluidframework/gitresources";
```

Comment:
Feels like there are no changes here over server version. It should probably be not in server version. These are just definitions, so probably fine to clone (if we are cloning implementations).
Just checking intention.

Reply:
Yes, I was thinking I can have a separate PR if we want to clone interfaces too. Just don't want to do everything in one PR. In this PR, I just want to clone what is required.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/4078